nncf/quantization/algo.py

```diff
@@ -495,18 +492,14 @@ def __init__(self, config, should_init: bool = True):
             self._use_logarithm_scale_per_group[quantizer_group] = params_dict.get('logarithm_scale', False)
 
         self._disable_saturation_fix = self.config.get('disable_saturation_fix', False)
+        # JSON-compatible dictionary that represents compression setup
+        self._quantizer_setup_state = {}  # type: Dict
```

Comment:
Would be good to have a specialized class object here instead of a dict, which may group together the related key storage constants and methods